src/cpp/session/include/session/SessionOptions.gen.hpp

```diff
@@ -456,21 +456,21 @@ class GeneratedOptions
    bool handleOfflineEnabled() const { return handleOfflineEnabled_; }
    int handleOfflineTimeoutMs() const { return handleOfflineTimeoutMs_; }
    bool sessionUseFileStorage() const { return sessionUseFileStorage_; }
-   bool allowVcsExecutableEdit() const { return allowVcsExecutableEdit_; }
-   bool allowCRANReposEdit() const { return allowCRANReposEdit_; }
-   bool allowVcs() const { return allowVcs_; }
-   bool allowPackageInstallation() const { return allowPackageInstallation_; }
-   bool allowShell() const { return allowShell_; }
-   bool allowTerminalWebsockets() const { return allowTerminalWebsockets_; }
-   bool allowFileDownloads() const { return allowFileDownloads_; }
-   bool allowFileUploads() const { return allowFileUploads_; }
-   bool allowRemovePublicFolder() const { return allowRemovePublicFolder_; }
-   bool allowRpubsPublish() const { return allowRpubsPublish_; }
-   bool allowExternalPublish() const { return allowExternalPublish_; }
-   bool allowPublish() const { return allowPublish_; }
-   bool allowPresentationCommands() const { return allowPresentationCommands_; }
-   bool allowFullUI() const { return allowFullUI_; }
-   bool allowLauncherJobs() const { return allowLauncherJobs_; }
+   bool allowVcsExecutableEdit() const { return allowVcsExecutableEdit_ || allowOverlay(); }
+   bool allowCRANReposEdit() const { return allowCRANReposEdit_ || allowOverlay(); }
+   bool allowVcs() const { return allowVcs_ || allowOverlay(); }
+   bool allowPackageInstallation() const { return allowPackageInstallation_ || allowOverlay(); }
+   bool allowShell() const { return allowShell_ || allowOverlay(); }
+   bool allowTerminalWebsockets() const { return allowTerminalWebsockets_ || allowOverlay(); }
+   bool allowFileDownloads() const { return allowFileDownloads_ || allowOverlay(); }
+   bool allowFileUploads() const { return allowFileUploads_ || allowOverlay(); }
```

Comment:
Doesn't `allowOverlay()` always return `false` in OS? In that case, what's the purpose of adding the ` || ` clause?